### ISSUE TITLE
Extend parser template to create an ELN-ready entry from a file

### ISSUE THREAD

**Issue by fabianschoeppach** [FAIRmat-NFDI/cookiecutter-nomad-plugin#41]
Hey, 

as we have already discussed last week, there is an urgent need for a template (or think of it as a best practice guide) on how to write an ELN-ready entry from a measurement file (e.g. a simple two-column csv file with possibly a header with metadata).
I think employing this in this plugin template would go long way. 

I think reading from files into ELN entries is THE thing in the experimental area of how data gets into Nomad, and unfortunately the current parser template does not help with that at all.

**Comment by blueraft** [comment]
In my (admittedly lay) view, instead of having this be part of the template, this sounds more like something that we should provide more examples of - similar to what's done [here](https://github.com/tokio-rs/axum/tree/main/examples). If area C is writing a parser, this measurements stuff is probably not very helpful then. What do you think @lauri-codes?

**Comment by fabianschoeppach** [comment]
Hey, thanks for the feedback!

I agree the current parser template is slanted toward simulation data and doesn’t address the needs of experimental workflows, which basically always require ELN capabilities for their entries. While I don’t know the balance of users working with simulation vs. experimental data, in my opinion experimentalists need better support. Right now, the parser template is rather useless for them.

Maybe it would be good to split the parser template into two types: Simulation Parser/no ELN-mode/as is and Experimental Parser/ELN-mode. This would better cater to both groups and make the templates more useful across the board. Thoughts?

**Comment by blueraft** [comment]
I agree that we should cater to experimentalists better here, just not sure if a more complicated template is the way to go here. The goal of the template was to have a very minimal package.

@hampusnasstrom and I discussed having more examples for different kind of plugins, which show cases plugin functionality at different difficulty levels. We could then have examples there for multiple categories of plugins (simulation/ experimental/ etc). It'd also be easier to maintain all of the examples in a separate repo.

**Comment by hampusnasstrom** [comment]
I'm not sure what you want to have in the template @fabianschoeppach? I think I agree with @blueraft that we should provide more examples.
